source/game/kart/CollisionGroup.hh

```diff
@@ -25,7 +26,8 @@ struct CollisionData {
     u32 closestFloorSettings;
     s32 intensity;
 
-    bool floor;
+    bool bFloor;
+    bool bSoftWall;
```

Comment:
Yessir, this is `CollisionType CollisionData::types` in Ghidra. It is a bitfield because it can take on multiple values (`SOFT_WALL` and `FLOOR` for example)

Reply:
Thought as much, just wanted to double check.